### PR TITLE
Switch to github-actions

### PR DESCRIPTION
**Pull request by zonky2**
Switch to github-actions

### CODE REVIEW

File: tests/DependencyInjection/MetaModelsFilterPerimetersearchExtensionTest.php [hunk 1]
```diff
@@ -28,6 +29,7 @@
 
 /**
  * This test case test the extension.
+ * @covers \MetaModels\FilterPerimetersearchBundle\DependencyInjection\MetaModelsAttributeRatingExtension
```
Comment (discordier) [inline tests/DependencyInjection/MetaModelsFilterPerimetersearchExtensionTest.php:32]: Please add a line break here (as for all other tests).

**Review by discordier — CHANGES_REQUESTED** [review]

File: tests/DependencyInjection/MetaModelsFilterPerimetersearchExtensionTest.php (hunk 1 above)
Comment (zonky2) [reply tests/DependencyInjection/MetaModelsFilterPerimetersearchExtensionTest.php:32]: done

Comment (discordier) [reply tests/DependencyInjection/MetaModelsFilterPerimetersearchExtensionTest.php:32]: Would you please push the change then? It is not here on github yet.

**Review by discordier — CHANGES_REQUESTED** [review]
Line break still missing.

File: tests/DependencyInjection/MetaModelsFilterPerimetersearchExtensionTest.php (hunk 1 above)
Comment (zonky2) [reply tests/DependencyInjection/MetaModelsFilterPerimetersearchExtensionTest.php]: pls check now

**Review by discordier — APPROVED** [review]

**Comment by zonky2** [comment]
Implemented in MM 2.2

**Pull request closed without merging**